tests/test_queries/test_order_by.py

```diff
@@ -163,6 +163,50 @@ async def test_sort_order_on_main_model():
         assert songs[2].name == "Song 2"
         assert songs[3].name == "Song 3"
 
+        await Song.objects.create(name="Song 5")
```

Comment:
There are missing tests for sorting by related model field (so sorting songs by album.title or whatever)